Bio/SeqFeature.py

```diff
@@ -1156,7 +1164,7 @@ def extract(self, parent_sequence, references=None):
                 parent_sequence = parent_sequence.seq
             except AttributeError:
                 pass
-        f_seq = parent_sequence[self.nofuzzy_start : self.nofuzzy_end]
+        f_seq = parent_sequence[int(self.start) : int(self.end)]
```

Comment:
I could construct pathological location/position objects where this differs from the no-fuzzy version, but can we make a GenBank location string which when parsed has that issue?

i.e. Using OneOfPosition, WithinPosition, or BetweenPosition